extension/pnpm-lock.yaml

```diff
@@ -0,0 +1,3695 @@
+lockfileVersion: '9.0'
```

Comment:
Can you remove lock file since we are using pnpm monorepo with one lock file for every subdir?